Sources/AppStoreConnectCLI/Services/Operations/ListBetaTestersOperation.swift

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBetaTestersOperation: APIOperation {
+
+    struct Options {
+        let email: String?
+        let firstName: String?
+        let lastName: String?
+        let inviteType: BetaInviteType?
+        let appIds: [String]?
+        let groupIds: [String]?
+        let sort: ListBetaTesters.Sort?
+        let limit: Int?
+        let relatedResourcesLimit: Int?
+    }
+
+    enum Error: LocalizedError {
+        case notFound
+
+        var errorDescription: String? {
+            switch self {
+            case .notFound:
+                return "Beta testers with provided filters not found."
+            }
+        }
+    }
+
+    var limits: [ListBetaTesters.Limit] {
```

Comment:
Probably best to make these computed `var`s that are used inside `execute` private